pubspec.yaml

```diff
@@ -2,10 +2,10 @@ name: lints
 description: Official Dart lint rules. Defines the 'core' and 'recommended' set of lints suggested by the Dart team.
 homepage: https://github.com/dart-lang/lints
 repository: https://github.com/dart-lang/lints
-version: 1.0.1
+version: 2.0.0
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.17.0-0 <3.0.0'
```

Comment:
Need to verify that this is the lower bound that we want.

Reply:
Since nobody has raised any concerns about this in the email thread, I am assuming this constraint is good.

Reply:
Using a non-stable lower bound is OK. It has the added benefit that we can publish prior to 2.17 stable going out, and pub.dev will show the package as a "pre-stable release" until the exact moment 2.17 stable launches.

But rather than using `-0` we should perhaps use the build number for the lowest release that supports all the lints that we're including here?